app/views/layouts/_navbar.html.erb

```diff
@@ -18,7 +18,7 @@
           <li class="nav-item dropdown">
             <img class="avatar dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false" src="https://kitt.lewagon.com/placeholder/users/ssaunier" />
             <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
-              <a class="dropdown-item" href="#">Action</a>
+              <a class="dropdown-item" href="#">Login</a>
```

Comment:
The href needs to be updated with the login path.
See the chapter "Generated routes" from https://kitt.lewagon.com/camps/1515/lectures/05-Rails%2F06-Airbnb-Devise 
Please use the route_name_path convention rather than typing in the url